rest/route/project_test.go

```diff
@@ -112,3 +113,111 @@ func (s *ProjectGetSuite) TestGetRecentVersions() {
 	s.NoError(err)
 	s.EqualError(getVersions.Parse(ctx, request), "400 (Bad Request): Invalid offset")
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// Tests project create route
+type ProjectCreateSuite struct {
+	sc *data.MockConnector
+	h  *projectCreateHandler
+
+	suite.Suite
+}
+
+func TestProjectCreateSuite(t *testing.T) {
+	suite.Run(t, new(ProjectCreateSuite))
+}
+
+func (s *ProjectCreateSuite) SetupTest() {
+	s.sc = &data.MockConnector{}
+	s.h = &projectCreateHandler{sc: s.sc}
+}
+
+func (s *ProjectCreateSuite) TestRoute() {
+	payloadBytes, _ := json.Marshal(map[string]interface{}{
+		"identifier":  "id",
+		"branch_name": "branch",
+	})
+
+	payload := bytes.NewBuffer(payloadBytes)
+
+	r, err := http.NewRequest(
+		"PUT",
+		"https://evergreen.mongodb.com/rest/v2/projects/",
```

Comment:
this should not reference the mongodb evergreen installation... use example.net just to be clear.